app/views/mentor_overview/index.html.haml

```diff
@@ -1,9 +1,12 @@
+= render partial: 'shared/header'
+:javascript
+  window.onbeforeunload = null;
 %h1 Webapp Begeleiders
 %p
   Voor
   %i
-    = @mentor.first_name
-    = @mentor.last_name
+    = @current_user.first_name
+    = @current_user.last_name
```

Comment:
Hier `current_user` van maken ipv `@current_user`? Volgens mij bestaat @current_user alleen als current_user eerder is aangeroepen in dit request.